src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -353,6 +386,23 @@ private static SecConditional getPrecedingConditional(List<TriggerItem> triggerI
 		return null;
 	}
 
+	private static List<SecConditional> getPrecedingConditionals(List<TriggerItem> triggerItems) {
+		List<SecConditional> conditionals = new ArrayList<>();
+		// loop through the triggerItems in reverse order so that we find the most recent items first
+		for (int i = triggerItems.size() - 1; i >= 0; i--) {
+			TriggerItem triggerItem = triggerItems.get(i);
+            if (!(triggerItem instanceof SecConditional))
+                break;
+			SecConditional conditional = (SecConditional) triggerItem;
+            if (conditional.type == ConditionalType.ELSE)
```

Comment:
formatting is messed up
```suggestion
		if (conditional.type == ConditionalType.ELSE)
```

Reply:
Not resolved 😔